test/f90_correct/src/precision00.f08

```diff
@@ -0,0 +1,38 @@
+! Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+! See https://llvm.org/LICENSE.txt for license information.
+! SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+
```

Comment:
single empty line (as in all other files) would do.

Reply:
Done.